--- vrpn/vrpn_Tracker_JsonNet.C.orig	2014-11-15 20:29:38.000000000 +0100
+++ vrpn/vrpn_Tracker_JsonNet.C	2014-12-26 12:45:58.731261102 +0100
@@ -16,7 +16,7 @@
 	#define INVALID_SOCKET -1
 #endif
 
-#include "json/json.h"
+#include <jsoncpp/json.h>
 
 #include "quat.h"
 
--- vrpn/cmake/FindJsonCpp.cmake.orig	2014-11-15 20:29:36.000000000 +0100
+++ vrpn/cmake/FindJsonCpp.cmake	2014-12-27 09:48:50.844747877 +0100
@@ -21,8 +21,9 @@
 	CACHE
 	PATH
 	"Directory to search for JSONCPP")
-set(_jsoncppnames)
+set(_jsoncppnames jsoncpp)
 set(_pathsuffixes
+	lib${LIB_SUFFIX}
 	suncc
 	vacpp
 	mingw
@@ -77,7 +78,7 @@
 
 find_path(JSONCPP_INCLUDE_DIR
 	NAMES
-	json/json.h
+	jsoncpp/json.h
 	PATHS
 	"${JSONCPP_ROOT_DIR}"
 	PATH_SUFFIXES
